x-pack/plugins/ingest_manager/common/types/epm.ts

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
can we nest this file under `types/models/`?
i.e. `x-pack/plugins/ingest_manager/common/types/models/epm.ts`